benchmark_terraform/modules/benchmark/service/init.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+## Enable strict error handling, command tracing, and pipefail
+set -euxo pipefail
+
+# Check if Docker is installed
+if ! command -v docker &> /dev/null; then
+    echo "Docker not found. Installing Docker..."
+    sudo dnf install -y docker docker-compose
```

Comment:
`compose` is already available as a subcommand of `docker`. See https://github.com/ChainSafe/forest-iac/issues/124